src/transformDimension.ts

```diff
@@ -61,9 +68,10 @@ export function transformDimension(token: DesignToken): DesignToken['value'] {
       transformed = transformProp(transformed, 'width');
       break;
     }
-    default:
-      transformed = transformed as number | string;
-      transformed = _transformDimension(transformed);
+    default: {
+      transformed = transformed as string | number;
+      transformed = splitMultiIntoSingle(transformed);
```

Comment:
Now that I'm looking at this again it seems like another wrapper function around these splitMultiIntoSingle calls makes sense for slightly better readability. This is already a lot better and it's certainly acceptable imo as it is now 👍🏻 . So also a nitpick, up to you